sig-cli/CONTRIBUTING.md

```diff
@@ -66,20 +73,7 @@ Since maintainers will need to own any code for these contributions, these shoul
 be very limited in scope and contain minimal risk
 (e.g. simple regression fixes, improved documentation, improved testing).
 
-## Before You Begin
-
-**Note**: Complete the following steps before reaching out to cli community members.
-
-### Agree to contribution rules
-
-Follow the [CLA signup instructions](../CLA.md).
```

Comment:
@nidorano why the CLA mention was removed?

Reply:
@idvoretskyi  Because it is part of the main contributor guide and we would like to discourage duplicate information.
A good way to do it if you don't want to remove it is to say "Remember to sign the CLA <link>". That way, it ties it in to the contrib guide.

Reply:
@idvoretskyi, as @guineveresaenger stated, the idea is to remove duplication of content. As the CLA is the only hard prerequisite for contributors, I will add a reminder in the first line.

Reply:
The required changes have been incorporated. @idvoretskyi, you can proceed with your review.